src/ProvidedTypes.fs

```diff
@@ -14477,7 +14477,7 @@ namespace ProviderImplementation.ProvidedTypes
                         match pminfo.GetInvokeCode with
                         | Some _ when ptdT.IsInterface ->
                             failwith "The provided type definition is an interface; therefore, it should not define an implementation for its members."
-                        | Some _ when pminfo.IsAbstract ->
+                        | Some _ when pminfo.IsAbstract && not pminfo.IsVirtual ->

```

Comment:
I'm not certain this is the right change. I think the right combination may require some combination of provided type definition properties and `pminfo` properties.